### DISCUSSION TITLE
Agregar una propiedad nativa para personalizar el estilo del Tooltip

### DISCUSSION THREAD

**Discussion by gonzaojedatn** [TiendaNube/nimbus-design-system#250] [Ideas] [answered]
Un partner nos consultó sobre cómo modificar el estilo del Tooltip, para evitar que se muestre a lo largo de toda la pantalla, tal como se observa en la siguiente imagen:
![image](https://github.com/user-attachments/assets/d8c0c598-5cd7-4d32-a33b-a7beb9e64df2)

El objetivo es que el Tooltip pueda adaptarse de manera más adecuada y estéticamente, como se muestra en la segunda imagen.

Tras revisar la documentación y realizar diversas pruebas, no encontré una propiedad `width` para el Tooltip que permita ajustar su estilización de la forma solicitada. 
Por este motivo, sugerí que utilice la biblioteca @floating-ui/react, que permite controlar y personalizar el comportamiento del Tooltip, logrando el diseño deseado por el partner:
![image](https://github.com/user-attachments/assets/38113081-61de-4b10-ad5b-d169edf62f70)

Dado que este tipo de personalización es una necesidad recurrente, sería útil incorporar una propiedad nativa que facilite la modificación del estilo del Tooltip sin tener que recurrir a soluciones externas.

**Comment by joacotornello** [comment, marked as answer]
Hola @gonzaojedatn, este issue fue resuelto en este [PR](https://github.com/TiendaNube/nimbus-design-system/pull/268), y liberado en las versiones:

```
@nimbus-ds/components@5.6.0
@nimbus-ds/styles@9.12.0
```

Se expuso una propiedad responsiva `maxWidth` y se mejoró la adaptabilidad del componente en pantallas mas chicas, eliminando el overflow.

Estamos a disposición por cualquier inquietud!
